pkg/kubectl/cmd/cmd.go

```diff
@@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 
 func checkErr(err error) {
```

Comment:
We're really trying to nuke this because it makes commands untestable in units.  Making it public is an implicit endorsement...


> On Mar 4, 2015, at 2:15 PM, Jeff Lowdermilk <notifications@github.com> wrote:
> 
> In pkg/kubectl/cmd/cmd.go:
> 
> > @@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
> >  
> >  func checkErr(err error) {
> I recall a few people, (@smarterclayton and others) saying this pattern is bad and we should discontinue it. I'm personally fine with the pattern if it actually does error parsing work instead of just wrapping logfatal.
> 
> That being said, there is a duplicate declaration of checkErr in pkg/cmd/util/helpers.go (mea culpa; I was lazy and didn't want to sed across all occurrences). IMO, it should just be declared in pkg/cmd/util. Please add a TODO to move it there, and I will do so after merging this.
> 
> —
> Reply to this email directly or view it on GitHub.
>